bitrise.yml

```diff
@@ -127,6 +128,31 @@ workflows:
         inputs:
         - verbose: 'false'
 
+  _sonarqube:
+    steps:
+    - script@1:
+        title: Setup env vars
+        inputs:
+        - content: |
+            #!/usr/bin/env bash
+
+            if [[ $BITRISE_GIT_BRANCH == release* ]]; then
+                envman add --key IS_RELEASE_BRANCH --value "true"
+            else
+                envman add --key IS_RELEASE_BRANCH --value "false"
+            fi
+    - sonarqube-scanner@1:
+        run_if: '{{getenv "SONAR_API_TOKEN" | ne "" | and (enveq "BITRISE_GIT_BRANCH" "master" | or (enveq "IS_RELEASE_BRANCH" "true") )}}'
+        inputs:
+        - scanner_properties: |-
+            sonar.sources=Sources
+            sonar.projectKey=$BITRISE_APP_TITLE
```

Comment:
Please add
```
sonar.projectName=iOS In-App Messaging SDK
```

I'm not sure how the internal projects have their name set to something different to the projectKey because internal Bitrise yaml isn't setting projectName 🤔

Reply:
👍 I've been wondering about that.
I'll also set explicit value for `sonar.projectKey`